### ISSUE TITLE
Detect flows outside of valid flow tag

### ISSUE THREAD

**Issue by OlofHaglund** [apigee/apigeelint#125]
If you write a flow statement outside of valid flow apigeelint lets it pass but Apigee will fail. Apigeelint should detect invalid flow statements.

For example according to apigeelint the following is valid:
```
<Flow name="Token">
    <Condition>(proxy.pathsuffix MatchesPath "/token") and (request.verb = "POST")</Condition>
    <Description>Trade code for token</Description>
    <Response/>
</Flow>
<Flow name="Default">
    <Description>Default, no valid flow found</Description>
    <Request>
        <Step>
            <Name>amSet405ErrorVariables</Name>
        </Step>
        <Step>
            <Name>fcRaiseDefaultError</Name>
        </Step>
    </Request>
    <Response/>
</Flow>
```
But it is no valid since it is not enclosed by a <Flows> tag.

Expected behavior: An error is raised.
Actual behavior: Config passes the lint.

**Comment by DinoChiesa** [comment]
Which part is not valid?  Which enclosing tag is missing?  Point it out more explicitly for me. Maybe provide a fuller (complete) example.

**Comment by DinoChiesa** [comment]
A check for misplaced Flow elements has now been included as part of the EP002 plugin, which has been merged in the past few days and is now available.
